smart-contracts/assembly/contracts/dns/__tests__/dns.spec.ts

```diff
@@ -125,23 +124,6 @@ describe('DNS contract tests', () => {
       setResolver(setResolverArgs);
     });
 
-    test('try to blackList a websiteName not being admin', () => {
-      switchUser(deployerAddress);
-
-      expect(() =>
-        addWebsiteToBlackList(new Args().add(name).serialize()),
-      ).toThrow();
-      expect(Storage.has(blackListKey)).toBeFalsy();
-    });
-
-    test('try to blackList a websiteName being admin', () => {
-      switchUser(dnsAdmin);
-      addWebsiteToBlackList(new Args().add(name).serialize());
-      expect(Storage.get(blackListKey)).toStrictEqual(
-        new Args().add(name).serialize(),
-      );
-    });
-
```

Comment:
why you delete these tests ? i think they are relevant. just use `addWebsitesToBlackList` instead of`addWebsiteToBlackList

Reply:
try to blackList a websiteName being admin is already covered , just added try to blackList a websiteName not being admin